package.json

```diff
@@ -55,6 +55,7 @@
     "fuse.js": "^3.2.0",
     "gettext-parser": "1.3.1",
     "grid-emotion": "^2.1.0",
+    "intersection-observer": "^0.7.0",
```

Comment:
Are we actually using intersection observer?

Reply:
@lynnagara yea. I'm using this for panning the minimap.